### ISSUE TITLE
Wheelchair icon doesn't do anything

### ISSUE THREAD

**Issue by mahutt** [mahutt/ViniMap#123]
### Summary
The accessibility mode in the mode-selector doesn't do anything. It should be removed if we do not yet have the functionality for it.

### Severity
Medium (Feature partially broken)

### Expected Behaviour
Pressing this button should include functionality for accesibility features.

### Actual Behaviour
Pressing button does not initiate any additional functionality

### Steps to Reproduce
1) Navigate to client directory
2) Run _npm run android_
3) Once the emulator is buitlt, attempt to press the accesibility mode selector

### Related Issues:
https://github.com/mahutt/ViniMap/issues/22
